carbonmark-api/test/routes/projects/get.test.ts

```diff
@@ -120,19 +132,18 @@ describe("GET /projects", () => {
     //@todo replace with composeEntries function
     const expectedResponse = [
       {
-        ...pick(digitalCarbon.digitalCarbonProject, ["region"]),
+        region: digitalCarbon.digitalCarbonProject.region,
         methodologies: [
           {
-            id: carbonProjects.carbonProject.methodologies[0].id,
-            category: carbonProjects.carbonProject.methodologies[0].category,
-            name: carbonProjects.carbonProject.methodologies[0].name,
+            id: mockCmsProject.methodologies?.[0]?.id,
+            category: mockCmsProject?.methodologies?.[0]?.category,
```

Comment:
nitpick: There is an inconsistent use of `?.` here (not used on the line above)

Reply:
ok, cleaned up